dbos/_utils.py

```diff
@@ -15,3 +18,31 @@ class GlobalParams:
     except importlib.metadata.PackageNotFoundError:
         # If package is not installed or during development
         dbos_version = "unknown"
+
+
+def retriable_postgres_exception(e: DBAPIError) -> bool:
```

Comment:
Looks like we can use `connection_invalidated` if it's a DBAPIError. https://docs.sqlalchemy.org/en/20/core/pooling.html#custom-legacy-pessimistic-ping

Reply:
https://github.com/sqlalchemy/sqlalchemy/blob/main/lib/sqlalchemy/dialects/postgresql/psycopg.py#L517-L521